### PR TITLE
small placeholder file and link to git repo api docs

### PR DESCRIPTION
**Pull request by EricDavisX**
Hello, this is a first part of support to fix / implement the Fleet API Docs per:
https://github.com/elastic/observability-docs/issues/881

I'm not 100% sure if I have all the bits in place to instantiate a new file and have it show up in the right hand side widget.  But I'll try it out.

2 subsequent prs (one to elasticsearch and one more to observability-docs) will follow to point to this new 'api docs' home page location.  we can change whats on the page anytime, but it seemed prudent to have it in place for google searchability and linking from other projects, etc.

### CODE REVIEW

**Comment by EricDavisX** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by EricDavisX** [comment]
@dedemorton @bmorelli25 hi. any thoughts why the docs guides aren't available?  I don't know how to re-trigger Jenkins to retry building them

**Comment by bmorelli25** [comment]
Hey @ericburns51! A 404 on the diff indicates one of two things: either the build didn't successfully run, or it didn't detect any changes. In this case, it appears to be the latter. The file you've added is not included anywhere in the build, so the change diff 404s. You'll need to add an include statement to the index file to get this new content built.

**Comment by dedemorton** [comment]
I'd include the link after the release notes in the Fleet User Guide.

File: docs/en/ingest-management/fleet/fleet-api-docs.asciidoc
Comment (dedemorton) [inline docs/en/ingest-management/fleet/fleet-api-docs.asciidoc:2]:
```suggestion
= {fleet} APIs
```

File: docs/en/ingest-management/fleet/fleet-api-docs.asciidoc
Comment (dedemorton) [inline docs/en/ingest-management/fleet/fleet-api-docs.asciidoc:12]:
```suggestion
```

Comment (dedemorton) [reply docs/en/ingest-management/fleet/fleet-api-docs.asciidoc:12]: page is not required because it doesn't use the tabbed widgets

**Comment by EricDavisX** [comment]
ah!  Thanks, I'll make an attempt now.

File: docs/en/ingest-management/fleet/fleet-api-docs.asciidoc
Comment (dedemorton) [inline docs/en/ingest-management/fleet/fleet-api-docs.asciidoc:5]:
```suggestion
{fleet} is fully supported by an API layer. Any actions you can perform
through the {fleet} UI are also available through the API. In fact, the UI
consumes the API.
```

File: docs/en/ingest-management/fleet/fleet-api-docs.asciidoc
Comment (dedemorton) [inline docs/en/ingest-management/fleet/fleet-api-docs.asciidoc:9]:
```suggestion
Please refer to the
https://github.com/elastic/kibana/blob/master/x-pack/plugins/fleet/common/openapi/README.md[{fleet} OpenAPI file] for more details.
```

File: docs/en/ingest-management/fleet/fleet-api-docs.asciidoc
Comment (EricDavisX) [reply docs/en/ingest-management/fleet/fleet-api-docs.asciidoc:2]: : / lol, I wasn't sure how this was expecting to resolve.  i couldn't have made this pr any more complicated (and wrong) for how small it is.

File: docs/en/ingest-management/fleet/fleet-api-docs.asciidoc
Comment (EricDavisX) [reply docs/en/ingest-management/fleet/fleet-api-docs.asciidoc:5]: if you don't prefer the language I used, it was copied from another page I think, I can look for it and update it there for us too.  Thanks for the rewrite.

Comment (dedemorton) [reply docs/en/ingest-management/fleet/fleet-api-docs.asciidoc]: @EricDavisX I think the language has a few problems. It's too vague because it doesn't qualify which UI we're talking about. Generally it's good to avoid words like "thing" because they are too broad and don't translate well. Also, the word "anything" tends to over-promise. The last line seems too enthusiastic for docs, but would probably be OK in the blog post.

File: docs/en/ingest-management/fleet/fleet-api-docs.asciidoc
Comment (dedemorton) [inline docs/en/ingest-management/fleet/fleet-api-docs.asciidoc:9]:
```suggestion
https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/fleet/common/openapi/README.md[{fleet} OpenAPI file] in the {kib} repo for more details.
```

Comment (dedemorton) [reply docs/en/ingest-management/fleet/fleet-api-docs.asciidoc:9]: Sorry there is one last thing I forgot to fix. We need to use a variable here, or the URL will always point to master and not necessarily reflect the correct version.

**Review by dedemorton — APPROVED** [review]
Marking as approved (pending acceptance of final suggestion), but please do not merge until we get confirmation from the dev team that we want to externalize this API officially. cc @andresrc for visibility

**Comment by EricDavisX** [comment]
the Guide looks as expected, it rendered now.  so cool.  thanks Dede and Brandon.  I'll wait on this as noted.

File: docs/en/ingest-management/fleet/fleet-api-docs.asciidoc
Comment (jen-huang) [inline docs/en/ingest-management/fleet/fleet-api-docs.asciidoc:6]:
```suggestion
through the {fleet} UI are also available through the API.
```

suggest to remove this last bit as well, that's how it generally works :)

**Review by jen-huang — APPROVED** [review]

File: docs/en/ingest-management/fleet/fleet-api-docs.asciidoc
Comment (EricDavisX) [reply docs/en/ingest-management/fleet/fleet-api-docs.asciidoc:6]: indeed.  :)

**Comment by EricDavisX** [comment]
Dede and Andres confirmed we're good to go here.

**Pull request merged**
